charts/atlantis/templates/statefulset.yaml

```diff
@@ -522,10 +522,17 @@ spec:
             mountPath: /home/atlantis/.netrc
             subPath: netrc
           {{- end }}
-          {{- if or .Values.aws.credentials .Values.aws.config .Values.awsSecretName }}
+          {{- range $filename, $_ := .Values.aws }}
+          {{- if has $filename (list "credentials" "config") }}
           - name: aws-volume
             readOnly: true
-            mountPath: {{ .Values.aws.directory | default "/home/atlantis/.aws" }}
+            mountPath: {{ $.Values.aws.directory | default "/home/atlantis/.aws" }}/{{ $filename }}
+            subPath: {{ $filename }}
+          {{- else if has $filename (list "awsSecretName") }}
```

Comment:
The key `awsSecretName` is not within `.Values.aws`, is something expected to change or it's a mistake?

Reply:
yeah just a mistake.  didnt notice it wasnt part of the aws block.